neuro_san/test/assessor/assessor_assert_forwarder.py

```diff
@@ -12,10 +12,10 @@
 from typing import Dict
 from typing import List
 
-from neuro_san.test.interfaces.assert_forwarder import AssertForwarder
+from neuro_san.test.interfaces.null_assert_forwarder import NullAssertForwarder
 
 
-class AssessorAssertForwarder(AssertForwarder):
+class AssessorAssertForwarder(NullAssertForwarder):
```

Comment:
Derive from the new NullAssertForwarder so this guy's business focuses on the Gist logic and not all the non-implementation of the asserts it doesn't care about.